Header header
uint32[] mode                         # mapping, repeating
Features[] map_features
Features[] live_features
Features[] map_transitions
float32[] map_similarity
float32[] map_offset
float32[] map_distances
time[] map_timestamps                 # timestamps of features
uint32 map_num                      # map_id, number of maps, closest img idx